app/containers/Applications/ApplicationCommentsByForm.tsx

```diff
@@ -13,8 +13,10 @@ interface Props {
 
 export const ApplicationCommentsByForm: React.FunctionComponent<Props> = props => {
   const {applicationReview} = props;
-  const reviewComments =
-    applicationReview.reviewCommentsByApplicationReviewId.edges;
+  const reviewComments = applicationReview
```

Comment:
You could use optional chaining here: 
```suggestion
  const reviewComments = applicationReview?.reviewCommentsByApplicationReviewId?.edges;
```